js/decentralizer.js

```diff
@@ -403,18 +417,39 @@ function initialLoad() {
                                     + '<td>'
                                         + (contracts[i].size/1000000000).toFixed(2) +' GB'
                                     + '</td>'
-                                    + '<td>'
-                                        + '<div class="tooltip"> ' + (contracts[i].totalcost/1000000000000000000000000).toFixed(2) + ' SC'
+                            
+                            // Value
+                            let totalSpent = (contracts[i].totalcost/1000000000000000000000000) - (contracts[i].renterfunds/1000000000000000000000000)
+                            tableContracts = tableContracts + '<td>'
+                                        + '<div class="tooltip"> ' + Math.round(totalSpent) + ' SC'
                                             + '<span class="tooltiptext">'
+                                                + 'Allocated: ' + (contracts[i].totalcost/1000000000000000000000000).toFixed(2) + ' SC<br>'
+                                                + 'Spent: ' + (totalSpent).toFixed(2) + ' SC<br>'
+                                                + 'Remaining: ' + (contracts[i].renterfunds/1000000000000000000000000).toFixed(2) + ' SC<br><br>'
                                                 + 'Storage spent: ' + (contracts[i].storagespending/1000000000000000000000000).toFixed(2) + ' SC<br>'
                                                 + 'Upload spent: ' + (contracts[i].uploadspending/1000000000000000000000000).toFixed(2) + ' SC<br>'
                                                 + 'Download spent: ' + (contracts[i].downloadspending/1000000000000000000000000).toFixed(2) + ' SC<br>'
                                                 + 'Fees: ' + (contracts[i].fees/1000000000000000000000000).toFixed(2) + ' SC<br>'
-                                                + 'Remaining: ' + (contracts[i].renterfunds/1000000000000000000000000).toFixed(2) + ' SC'
                                             + '</span>'			
                                         + '</div>'
                                     + '</td>'
 
+                            // Storageprice
+                            tableContracts = tableContracts + '<td>' + Math.round(contracts[i].storageprice * 400 / 92592592592) + ' SC</td>'

```

Comment:
where come from `400` and `92592592592`?

Reply:
From Hakkane 😁😁

What you think, is eatimated price and cost useful? Are they need 2 columns, or just hide the estimated data under a tooltip?
What about other prices (upload/download)? I don't want to make the table too big, but I miss them

Reply:
For me storage cost is the most helpful.

Reply:
What you think, is the old Value column was useful for you, or the Spend col is better? (for me, contract value does not makes sense, that's why I changed it to Spent)

Reply:
I had use it by dividing it by the current storage, so that I can exactly see how much it cost